cluster/saltbase/salt/kubelet/default

```diff
@@ -15,6 +15,15 @@
   {% set api_servers = "--api_servers=https://" + ips[0][0] + ":6443" -%}
 {% endif -%}
 
+# Disable registration for the kubelet running on the master on GCE.
```

Comment:
Robert, fyi I tested your kubelet mod in my GCE environment this weekend.
Worked perfectly! I deploy nodes with cloud configs derived from Paulo's
and the nodes show up with proper names that I can then use to create
services with an external load balancer.

Very nice, Thanks ... WkH

On Fri, Apr 17, 2015 at 8:19 PM, Robert Bailey <notifications@github.com>
wrote:

> In cluster/saltbase/salt/kubelet/default
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/6949#discussion_r28639946>
> :
>
> > @@ -15,6 +15,15 @@
> >    {% set api_servers = "--api_servers=https://" + ips[0][0] + ":6443" -%}
> >  {% endif -%}
> >
> > +# Disable registration for the kubelet running on the master on GCE.
>
> It keeps things consistent with the current deployment. I'd be happy to
> change this in a future PR to be configurable, but didn't want too many
> moving pieces here. For one, we'd need to change cluster validation to
> expect N+1 nodes (rather than N) since the master is now counted. And I
> didn't want to worry about other ramifications at the same time as changing
> the node creation pattern.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/6949/files#r28639946>
> .
>



-- 
... WkH